internal/hud/renderer.go

```diff
@@ -217,22 +223,27 @@ func bestLogs(res view.Resource) string {
 		return string(res.LastBuild().Log)
 	}
 
-	// Two cases:
-	// 1) The last build finished before this pod started
-	// 2) This log is from an in-place container update.
-	// in either case, prepend them to pod logs.
-	if (res.LastBuild().StartTime.Equal(res.PodUpdateStartTime) ||
-		res.LastBuild().StartTime.Before(res.PodCreationTime)) &&
-		len(res.LastBuild().Log) > 0 {
-		return string(res.LastBuild().Log) + "\n" + res.PodLog
-	}
+	// TODO(matt) figure out what to do with DC logs once we have DC timestamps
+
+	if res.IsK8S() {
```

Comment:
i'd do `if kri := m.K8sInfo(); !kri.Empty() {...}` to save a line, but 🤷‍♀️ 
also that stands for... kubernetes resource info? i don't care much but it's inconsistent with naming scheme i used for . `dcInfo`.

Reply:
I didn't implement `Empty` for k8sInfo, mostly because I've got an engrained bias against reflection that hasn't proven necessary, but maybe I'm just being squeamish and that's premature optimization.

Anyway, per your other [comment](https://github.com/windmilleng/tilt/pull/917#discussion_r245344865), I'll change other things to match this.